eva-accession-core/src/test/resources/input-files/assembly-report/AssemblyReport.txt

```diff
@@ -0,0 +1,67 @@
+# Assembly name:  GRCm38.p6
```

Comment:
Could we make this filename a bit more specific? Chicken for instance contains the assembly accession and name.